sick_test_server:
  ros__parameters:
    # ros parameters
    scanner_name : "sick_ldmrs"
    port : 12002
    send_scan_data_rate : 20.0 # frequency to generate and send scan data (default: 20 Hz)
    scan_data_payload: [ 0xa8, 0x03, 0x0b, 0x03, 0x00, 0x00, 0x38, 0xc3,
      0xce, 0x17, 0xa0, 0x00, 0x00, 0x00, 0x0c, 0xff, 0x7c, 0x1d, 0xa0, 0x00, 0x00, 0x00, 0x00, 0x2d,
      0x40, 0x06, 0xc0, 0xf9, 0xe4, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
      0x00, 0x00, 0x02, 0x00, 0x00, 0x50, 0x40, 0x06, 0x7d, 0x00, 0x90, 0x00, 0x00, 0x00, 0x01, 0x50,
      0x40, 0x06, 0x7d, 0x00, 0xa8, 0x00, 0x00, 0x00, 0x00, 0x44, 0x30, 0x06, 0x7e, 0x00, 0xac, 0x00,
      0x00, 0x00, 0x01, 0x44, 0x30, 0x06, 0x81, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x00, 0x50, 0x20, 0x06,
      0x78, 0x00, 0x98, 0x00, 0x00, 0x00, 0x01, 0x54, 0x20, 0x06, 0x83, 0x00, 0xb8, 0x00, 0x00, 0x00,
      0x00, 0x44, 0x10, 0x06, 0x82, 0x00, 0xb8, 0x00, 0x00, 0x00, 0x01, 0x44, 0x10, 0x06, 0x85, 0x00,
      0xd0, 0x00, 0x00, 0x00, 0x00, 0x54, 0x00, 0x06, 0x83, 0x00, 0xb0, 0x00, 0x00, 0x00, 0x01, 0x54,
      0x00, 0x06, 0x85, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x00, 0x40, 0xf0, 0x05, 0x84, 0x00, 0xc0, 0x00,
      0x00, 0x00, 0x01, 0x44, 0xf0, 0x05, 0x85, 0x00, 0xd0, 0x00, 0x00, 0x00, 0x00, 0x54, 0xe0, 0x05,
      0x85, 0x00, 0xb8, 0x00, 0x00, 0x00, 0x01, 0x54, 0xe0, 0x05, 0x86, 0x00, 0xc8, 0x00, 0x00, 0x00,
      0x00, 0x40, 0xd0, 0x05, 0x86, 0x00, 0xc8, 0x00, 0x00, 0x00, 0x01, 0x44, 0xd0, 0x05, 0x87, 0x00,
      0xd8, 0x00, 0x00, 0x00, 0x00, 0x54, 0xc0, 0x05, 0x85, 0x00, 0xbc, 0x00, 0x00, 0x00, 0x01, 0x54,
      0xc0, 0x05, 0x88, 0x00, 0xd0, 0x00, 0x00, 0x00, 0x00, 0x44, 0xb0, 0x05, 0x82, 0x00, 0xc8, 0x00,
      0x00, 0x00, 0x01, 0x44, 0xb0, 0x05, 0x83, 0x00, 0xd8, 0x00, 0x00, 0x00, 0x00, 0x54, 0xa0, 0x05,
      0x87, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x54, 0xa0, 0x05, 0x8e, 0x00, 0xc8, 0x00, 0x00, 0x00,
      0x00, 0x44, 0x90, 0x05, 0x8e, 0x00, 0xc8, 0x00, 0x00, 0x00, 0x01, 0x44, 0x90, 0x05, 0x8f, 0x00,
      0xd8, 0x00, 0x00, 0x00, 0x00, 0x54, 0x80, 0x05, 0x87, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x50,
      0x80, 0x05, 0x8e, 0x00, 0xc8, 0x00, 0x00, 0x00, 0x00, 0x40, 0x70, 0x05, 0x8e, 0x00, 0xc8, 0x00,
      0x00, 0x00, 0x01, 0x44, 0x70, 0x05, 0x8f, 0x00, 0xd8, 0x00, 0x00, 0x00, 0x00, 0x54, 0x60, 0x05,
      0x8f, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x54, 0x60, 0x05, 0x92, 0x00, 0xc8, 0x00, 0x00, 0x00,
      0x00, 0x44, 0x50, 0x05, 0x8e, 0x00, 0xc8, 0x00, 0x00, 0x00, 0x01, 0x44, 0x50, 0x05, 0x8f, 0x00,
      0xd8, 0x00, 0x00, 0x00, 0x00, 0x54, 0x40, 0x05, 0x8d, 0x00, 0xb8, 0x00, 0x00, 0x00, 0x01, 0x54,
      0x40, 0x05, 0x8e, 0x00, 0xc8, 0x00, 0x00, 0x00, 0x00, 0x44, 0x30, 0x05, 0x8c, 0x00, 0xc4, 0x00,
      0x00, 0x00, 0x01, 0x44, 0x30, 0x05, 0x8f, 0x00, 0xd8, 0x00, 0x00, 0x00, 0x00, 0x54, 0x20, 0x05,
      0x87, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x54, 0x20, 0x05, 0x8e, 0x00, 0xc8, 0x00, 0x00, 0x00,
      0x00, 0x44, 0x10, 0x05, 0x8c, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x44, 0x10, 0x05, 0x8d, 0x00,
      0xd4, 0x00, 0x00, 0x00, 0x00, 0x54, 0x00, 0x05, 0x8d, 0x00, 0xb8, 0x00, 0x00, 0x00, 0x01, 0x54,
      0x00, 0x05, 0x8e, 0x00, 0xc8, 0x00, 0x00, 0x00, 0x00, 0x44, 0xf0, 0x04, 0x8c, 0x00, 0xc4, 0x00,
      0x00, 0x00, 0x01, 0x44, 0xf0, 0x04, 0x8f, 0x00, 0xd8, 0x00, 0x00, 0x00, 0x00, 0x54, 0xe0, 0x04,
      0x93, 0x00, 0xb0, 0x00, 0x00, 0x00, 0x01, 0x50, 0xe0, 0x04, 0x95, 0x00, 0xc4, 0x00, 0x00, 0x00,
      0x00, 0x44, 0xd0, 0x04, 0x96, 0x00, 0xc8, 0x00, 0x00, 0x00, 0x01, 0x44, 0xd0, 0x04, 0x95, 0x00,
      0xd0, 0x00, 0x00, 0x00, 0x00, 0x54, 0xc0, 0x04, 0x97, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x54,
      0xc0, 0x04, 0x96, 0x00, 0xc8, 0x00, 0x00, 0x00, 0x00, 0x44, 0xb0, 0x04, 0x98, 0x00, 0xd0, 0x00,
      0x00, 0x00, 0x01, 0x44, 0xb0, 0x04, 0x97, 0x00, 0xdc, 0x00, 0x00, 0x00, 0x00, 0x54, 0xa0, 0x04,
      0x97, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x54, 0xa0, 0x04, 0x96, 0x00, 0xcc, 0x00, 0x00, 0x00,
      0x00, 0x44, 0x90, 0x04, 0x94, 0x00, 0xd0, 0x00, 0x00, 0x00, 0x01, 0x44, 0x90, 0x04, 0x97, 0x00,
      0xdc, 0x00, 0x00, 0x00, 0x00, 0x54, 0x80, 0x04, 0x97, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x54,
      0x80, 0x04, 0x96, 0x00, 0xc8, 0x00, 0x00, 0x00, 0x00, 0x44, 0x70, 0x04, 0x90, 0x00, 0xd0, 0x00,
      0x00, 0x00, 0x01, 0x44, 0x70, 0x04, 0x91, 0x00, 0xe0, 0x00, 0x00, 0x00, 0x00, 0x54, 0x60, 0x04,
      0x91, 0x00, 0xbc, 0x00, 0x00, 0x00, 0x01, 0x54, 0x60, 0x04, 0x96, 0x00, 0xc8, 0x00, 0x00, 0x00,
      0x02, 0x54, 0x58, 0x04, 0x98, 0x00, 0x50, 0x00, 0x00, 0x00, 0x03, 0x54, 0x58, 0x04, 0x93, 0x00,
      0x7c, 0x00, 0x00, 0x00, 0x00, 0x44, 0x50, 0x04, 0x90, 0x00, 0xd0, 0x00, 0x00, 0x00, 0x01, 0x44,
      0x50, 0x04, 0x8f, 0x00, 0xdc, 0x00, 0x00, 0x00, 0x02, 0x44, 0x48, 0x04, 0x96, 0x00, 0x80, 0x00,
      0x00, 0x00, 0x03, 0x44, 0x48, 0x04, 0x91, 0x00, 0x94, 0x00, 0x00, 0x00, 0x00, 0x54, 0x40, 0x04,
      0x8f, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x54, 0x40, 0x04, 0x92, 0x00, 0xc8, 0x00, 0x00, 0x00,
      0x02, 0x50, 0x38, 0x04, 0x93, 0x00, 0x70, 0x00, 0x00, 0x00, 0x03, 0x54, 0x38, 0x04, 0x92, 0x00,
      0x84, 0x00, 0x00, 0x00, 0x00, 0x44, 0x30, 0x04, 0x90, 0x00, 0xd0, 0x00, 0x00, 0x00, 0x01, 0x44,
      0x30, 0x04, 0x8f, 0x00, 0xd8, 0x00, 0x00, 0x00, 0x02, 0x40, 0x28, 0x04, 0x92, 0x00, 0x8c, 0x00,
      0x00, 0x00, 0x03, 0x40, 0x28, 0x04, 0x97, 0x00, 0xa0, 0x00, 0x00, 0x00, 0x00, 0x50, 0x20, 0x04,
      0x8f, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x50, 0x20, 0x04, 0x92, 0x00, 0xcc, 0x00, 0x00, 0x00,
      0x02, 0x50, 0x18, 0x04, 0x93, 0x00, 0x7c, 0x00, 0x00, 0x00, 0x03, 0x50, 0x18, 0x04, 0x91, 0x00,
      0x98, 0x00, 0x00, 0x00, 0x00, 0x44, 0x10, 0x04, 0x90, 0x00, 0xd0, 0x00, 0x00, 0x00, 0x01, 0x44,
      0x10, 0x04, 0x8f, 0x00, 0xd8, 0x00, 0x00, 0x00, 0x02, 0x40, 0x08, 0x04, 0x8f, 0x00, 0x9c, 0x00,
      0x00, 0x00, 0x03, 0x44, 0x08, 0x04, 0x90, 0x00, 0xac, 0x00, 0x00, 0x00, 0x00, 0x54, 0x00, 0x04,
      0x87, 0x00, 0xc4, 0x00, 0x00, 0x00, 0x01, 0x54, 0x00, 0x04, 0x8e, 0x00, 0xcc, 0x00, 0x00, 0x00,
      0x02, 0x54, 0xf8, 0x03, 0x93, 0x00, 0x7c, 0x00, 0x00, 0x00, 0x03, 0x54, 0xf8, 0x03, 0x91, 0x00,
      0x98, 0x00, 0x00, 0x00, 0x00, 0x40, 0xf0, 0x03, 0x8a, 0x00, 0xcc, 0x00, 0x00, 0x00, 0x01, 0x44,
      0xf0, 0x03, 0x8b, 0x00, 0xdc, 0x00, 0x00, 0x00, 0x02, 0x44, 0xe8, 0x03, 0x94, 0x00, 0xa0, 0x00,
      0x00, 0x00, 0x03, 0x44, 0xe8, 0x03, 0x93, 0x00, 0xb0, 0x00, 0x00, 0x00, 0x00, 0x54, 0xe0, 0x03,
      0x87, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x54, 0xe0, 0x03, 0x8c, 0x00, 0xd0, 0x00, 0x00, 0x00,
      0x02, 0x54, 0xd8, 0x03, 0x92, 0x00, 0x80, 0x00, 0x00, 0x00, 0x03, 0x54, 0xd8, 0x03, 0x8e, 0x00,
      0xa0, 0x00, 0x00, 0x00, 0x00, 0x44, 0xd0, 0x03, 0x88, 0x00, 0xd0, 0x00, 0x00, 0x00, 0x01, 0x44,
      0xd0, 0x03, 0x87, 0x00, 0xdc, 0x00, 0x00, 0x00, 0x02, 0x44, 0xc8, 0x03, 0x8f, 0x00, 0x9c, 0x00,
      0x00, 0x00, 0x03, 0x44, 0xc8, 0x03, 0x8f, 0x00, 0xb0, 0x00, 0x00, 0x00, 0x00, 0x54, 0xc0, 0x03,
      0x87, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x54, 0xc0, 0x03, 0x8c, 0x00, 0xd0, 0x00, 0x00, 0x00,
      0x02, 0x54, 0xb8, 0x03, 0x8f, 0x00, 0x88, 0x00, 0x00, 0x00, 0x03, 0x54, 0xb8, 0x03, 0x8e, 0x00,
      0xa0, 0x00, 0x00, 0x00, 0x00, 0x44, 0xb0, 0x03, 0x88, 0x00, 0xd0, 0x00, 0x00, 0x00, 0x01, 0x44,
      0xb0, 0x03, 0x87, 0x00, 0xdc, 0x00, 0x00, 0x00, 0x02, 0x44, 0xa8, 0x03, 0x94, 0x00, 0xa0, 0x00,
      0x00, 0x00, 0x03, 0x44, 0xa8, 0x03, 0x8f, 0x00, 0xb4, 0x00, 0x00, 0x00, 0x00, 0x54, 0xa0, 0x03,
      0x87, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x54, 0xa0, 0x03, 0x88, 0x00, 0xd0, 0x00, 0x00, 0x00,
      0x02, 0x54, 0x98, 0x03, 0x8f, 0x00, 0x88, 0x00, 0x00, 0x00, 0x03, 0x54, 0x98, 0x03, 0x8e, 0x00,
      0xa0, 0x00, 0x00, 0x00, 0x00, 0x44, 0x90, 0x03, 0x82, 0x00, 0xcc, 0x00, 0x00, 0x00, 0x01, 0x44,
      0x90, 0x03, 0x83, 0x00, 0xdc, 0x00, 0x00, 0x00, 0x02, 0x44, 0x88, 0x03, 0x90, 0x00, 0xa4, 0x00,
      0x00, 0x00, 0x03, 0x44, 0x88, 0x03, 0x8f, 0x00, 0xb0, 0x00, 0x00, 0x00, 0x00, 0x54, 0x80, 0x03,
      0x87, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x54, 0x80, 0x03, 0x88, 0x00, 0xd0, 0x00, 0x00, 0x00,
      0x02, 0x54, 0x78, 0x03, 0x8b, 0x00, 0x90, 0x00, 0x00, 0x00, 0x03, 0x54, 0x78, 0x03, 0x8e, 0x00,
      0xa0, 0x00, 0x00, 0x00, 0x00, 0x44, 0x70, 0x03, 0x88, 0x00, 0xd0, 0x00, 0x00, 0x00, 0x01, 0x44,
      0x70, 0x03, 0x87, 0x00, 0xdc, 0x00, 0x00, 0x00, 0x02, 0x44, 0x68, 0x03, 0x8c, 0x00, 0xa4, 0x00,
      0x00, 0x00, 0x03, 0x44, 0x68, 0x03, 0x8f, 0x00, 0xb0, 0x00, 0x00, 0x00, 0x00, 0x54, 0x60, 0x03,
      0x87, 0x00, 0xc0, 0x00, 0x00, 0x00, 0x01, 0x54, 0x60, 0x03, 0x88, 0x00, 0xd0, 0x00, 0x00, 0x00,
      0x02, 0x54, 0x58, 0x03, 0x8b, 0x00, 0x90, 0x00, 0x00, 0x00, 0x03, 0x54, 0x58, 0x03, 0x8a, 0x00,
      0xa4, 0x00, 0x00, 0x00, 0x00, 0x44, 0x50, 0x03, 0x86, 0x00, 0xcc, 0x00, 0x00, 0x00, 0x01, 0x44,
      0x50, 0x03, 0x87, 0x00, 0xdc, 0x00, 0x00, 0x00, 0x02, 0x44, 0x48, 0x03, 0x8c, 0x00, 0xa4, 0x00,
      0x00, 0x00, 0x03, 0x44, 0x48, 0x03, 0x8f, 0x00, 0xb4, 0x00, 0x00, 0x00, 0x00, 0x50, 0x40, 0x03,
      0x85, 0x00, 0xb8, 0x00, 0x00, 0x00, 0x01, 0x54, 0x40, 0x03, 0x88, 0x00, 0xd0, 0x00, 0x00, 0x00,
      0x02, 0x54, 0x38, 0x03, 0x8b, 0x00, 0x90, 0x00, 0x00, 0x00, 0x03, 0x54, 0x38, 0x03, 0x8b, 0x00,
      0xa8, 0x00, 0x00, 0x00, 0x00, 0x44, 0x30, 0x03, 0x8c, 0x00, 0xc4, 0x00, 0x00, 0x00, 0x01, 0x40,
      0x30, 0x03, 0x8b, 0x00, 0xd8, 0x00, 0x00, 0x00, 0x02, 0x44, 0x28, 0x03, 0x8c, 0x00, 0xa4, 0x00,
      0x00, 0x00, 0x03, 0x44 ]
        